modules/Galleries/GalleriesList.module.scss

```diff
@@ -1,5 +1,5 @@
 .container {
-    padding: 0;
+    padding: $spacing-8 0 0;
```

Comment:
This spacing is not consistent with the spacing on the Category page. I think the spacing for categoryList should be updated in the `InfiniteStories` component

Reply:
updated